app/scripts/lib/fxa-client.js

```diff
@@ -335,6 +335,10 @@ define(function (require, exports, module) {
       return client.sessionDestroy(sessionToken);
     }),
 
+    sessionDestroy: withClient((client, sessionToken, options = {}) => {
```

Comment:
I know we haven't been very strict about this in this file, can you add JSDocs to these new methods? I've been adding them to new functions as I go.